NOTICE

```diff
@@ -1,2 +1,29 @@
 Amplify for iOS
 Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+
+**********************
+THIRD PARTY COMPONENTS
+**********************
+
+** SQLite.swift; version 0.12.2 -- https://github.com/stephencelis/SQLite.swift
```

Comment:
```suggestion
** SQLite.swift -- https://github.com/stephencelis/SQLite.swift
```
Unless the license changed in a different version, it would likely be better not to include the version here because it would need to be updated every time we update the dependency version and could easily be missed.

Reply:
we need to keep this and update attribution when version changes.